Makefile

```diff
@@ -28,11 +28,20 @@ build: ## Build the containers
 	${DOCKER_COMPOSE} build
 
 start: ## Start the containers
+	$(MAKE) db
+	$(MAKE) wait-healthy
+	$(MAKE) initdb
 	${DOCKER_COMPOSE} up --detach
 
 stop: ## Stop the containers
 	${DOCKER_COMPOSE} down
 
+db: ## Start the database container
+	${DOCKER_COMPOSE} up --detach db
+
+initdb: ## Setup database tables
```

Comment:
The comments add this to the list of commands on `make help`, but we probably don't want that (so should be like `run`).